src/Native/Runtime/regdisplay.h

```diff
@@ -75,6 +105,9 @@ struct REGDISPLAY
                           // for EH, but not adjusted, so we only need
                           // their values, not their addresses
 
+    ArmNonVolatileContextPointer *pCurrentContextPointers;
+    ArmVolatileContextPointer     volatileCurrContextPointers;
```

Comment:
This is not needed. The unwinder is works on Windows ARM - Unix ARM should be no different.